recipes/skll/meta.yaml

```
@@ -0,0 +1,90 @@
+{% set name = "skll" %}
+{% set version = "2.5" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/EducationalTestingService/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 5d690556a72a729b4039d50010ddf7470c68d97505e8b6b6f1a5cc29d2b0ae01
+
+build:
+  noarch: python
+  number: 0
+  script:  
+    - cd $SRC_DIR
+    - "{{ PYTHON }} -m pip install . --no-deps -vv"
+  
+  entry_points:
+    - compute_eval_from_predictions = skll.utils.commandline.compute_eval_from_predictions:main
+    - filter_features = skll.utils.commandline.filter_features:main
+    - generate_predictions = skll.utils.commandline.generate_predictions:main
+    - join_features = skll.utils.commandline.join_features:main
+    - plot_learning_curves = skll.utils.commandline.plot_learning_curves:main
+    - print_model_weights = skll.utils.commandline.print_model_weights:main
+    - run_experiment = skll.utils.commandline.run_experiment:main
+    - skll_convert = skll.utils.commandline.skll_convert:main
+    - summarize_results = skll.utils.commandline.summarize_results:main
+    - compute_eval_from_predictions = skll.utils.commandline.compute_eval_from_predictions:main
+    - filter_features = skll.utils.commandline.filter_features:main
+    - generate_predictions = skll.utils.commandline.generate_predictions:main
+    - join_features = skll.utils.commandline.join_features:main
+    - plot_learning_curves = skll.utils.commandline.plot_learning_curves:main
+    - print_model_weights = skll.utils.commandline.print_model_weights:main
+    - run_experiment = skll.utils.commandline.run_experiment:main
+    - skll_convert = skll.utils.commandline.skll_convert:main
+    - summarize_results = skll.utils.commandline.summarize_results:main
```

Looks like an accidental copy/paste:
```suggestion
  entry_points:
    - compute_eval_from_predictions = skll.utils.commandline.compute_eval_from_predictions:main
    - filter_features = skll.utils.commandline.filter_features:main
    - generate_predictions = skll.utils.commandline.generate_predictions:main
    - join_features = skll.utils.commandline.join_features:main
    - plot_learning_curves = skll.utils.commandline.plot_learning_curves:main
    - print_model_weights = skll.utils.commandline.print_model_weights:main
    - run_experiment = skll.utils.commandline.run_experiment:main
    - skll_convert = skll.utils.commandline.skll_convert:main
    - summarize_results = skll.utils.commandline.summarize_results:main
```

---

Yes, thank you!